tests_metricflow/snapshots/test_mf_engine.py/list/test_list_metrics__result0.txt

```diff
@@ -12,6 +12,7 @@ docstring:
   'average_booking_value',
   'average_instant_booking_value',
   'bookers',
+  'bookers_fill_nulls_with_0_join_to_timespine',
```

Comment:
Need to filter these out specifically for metrics in the APIs, not group by metrics (which aren't exposed in the APIs)